piwik-sdk/src/test/java/org/piwik/sdk/TrackerTest.java

```diff
@@ -248,7 +227,7 @@ public void testGetResolution() throws Exception {
         Tracker tracker = createTracker();
         TrackMe trackMe = new TrackMe();
         tracker.track(trackMe);
-        assertTrue(trackMe.build().contains("res=480x800"));
+        assertTrue(trackMe.build(), trackMe.build().contains("res=480x800"));
```

Comment:
Yeah, my bad. I mixed it up with [python's one](https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertTrue) which I use on a daily basis..